docs/tutorials/using-jsx-in-apostrophe.md

```diff
@@ -195,6 +200,7 @@ At the top of this file we are importing both `react` and the `createRoot` funct
 The remainder of this file is a standard widget player. This player is attached to the `[data-react-weather-widget]` attribute that we will need to add to the widget Nunjucks template. Within that element, it selects an element with an id of `react-weather-root` to create the root for our React component. We are also passing a prop we are getting from the `data-default-city` attribute on our `rootElement`. We will need to add this root element and the data into the data attribute that we will be getting from a schema field.
```

Comment:
Link to widget player docs

Reply:
Something about the last sentence seems confusing but it's hard to put my finger on it

Reply:
Added and re-phrased.